build.gradle

```diff
@@ -212,6 +212,7 @@ project('segmentstore:storage:impl') {
         compile project(':shared:metrics')
         testCompile project(':test:testcommon')
         testCompile project(path:':segmentstore:storage', configuration:'testRuntime')
+        testCompile group: 'io.netty', name: 'netty-tcnative-boringssl-static', version: '2.0.6.Final'
```

Comment:
where is this being used?

Reply:
For setting up SSL for secure BK server for test.

Reply:
I don't see any new imports or references to this anywhere.

Reply:
Removed the dependency.